core/authority_discovery/publisher/address_publisher.cpp

```diff
@@ -117,6 +117,26 @@ namespace kagome::authority_discovery {
       return outcome::success();
     }
 
+    OUTCOME_TRY(
+        raw,
+        audiEncode(ed_crypto_provider_,
+                   sr_crypto_provider_,
+                   *libp2p_key_,
+                   *libp2p_key_pb_,
+                   peer_info,
+                   *audi_key,
+                   std::chrono::system_clock::now().time_since_epoch()));
```

Comment:
steady_clock might work better here to avoid possible time moving error and keep it monotonic

Reply:
No, this must be absolute time, because we sent this to other computers.

https://en.cppreference.com/w/cpp/chrono/steady_clock